code/_onclick/item_attack.dm

```diff
@@ -39,6 +39,16 @@
 		visible_message("<span class='danger'>[attack_message]</span>",
 		"<span class='userdanger'>[attack_message]</span>")
 
+	if((butcher_results) && (stat == DEAD))
+		user.changeNext_move(CLICK_CD_MELEE)

```

Comment:
should put this after the if(sharpness), otherwise attacking corpses has an extra delay, which is kinda weird.